.github/actions/push-image-aws-azure/action.yml

```diff
@@ -0,0 +1,71 @@
+name: 'Push image to Amazon ECR and Azure ACR'
+description: 'Push image to Amazon ECR and Azure ACR'
```

Comment:
Spravil som samostatnu action z uploadu image do ECR, ACR.

V Buffer service bude image pre API a Worker + image pre Templates API:
Tj. bude to pouzite na 3 miestach.

Original:
https://github.com/keboola/keboola-as-code/blob/fe5f1bff3bec0e5bb3251394e9e25925e5f1122b/.github/workflows/push.yml#L634-L687